### ISSUE TITLE
🏗️ Check and add `display: inline-block` in `extraProperties` of `presetIcons`

### ISSUE THREAD

**Issue by jd-solanki** [jd-solanki/anu#9]
**Why**

Right now we have to add `display: inline-block` to icon if icon doesn't have flex parent and icon is rendered via `i` tag. e.g. `<i class="inline-block i-bx-home" />`

If we don't add `inline-block` class icon won't appear because its size will be 0x0.

**Process**

- [ ] Check if adding `display: inline-block` to all icons is safe and doesn't create any side effects
- [ ] Add `'display': 'inline-block'` to `extraProperties` in `presetIcons`
- [ ] Check the UI to confirm the changes & side effects
- [ ] Add the relative changes to the [installation docs](https://anu-vue.netlify.app/guide/getting-started/installation.html#usage)

**Comment by zguolee** [comment]
Maybe we can add a new icon component, like [NuxtUi](https://github.com/nuxt/ui/blob/main/packages/ui/src/components/NIcon.vue), so that we do not need to add `inline-block` in every place.

**Comment by jd-solanki** [comment]
> Maybe we can add a new icon component, like [NuxtUi](https://github.com/nuxt/ui/blob/main/packages/ui/src/components/NIcon.vue?rgh-link-date=2022-09-17T08%3A03%3A02Z), so that we do not need to add `inline-block` in every place.

Yes we can (_IG we might need it in the future_), but why not directly add it in `extraProperties` of `presetIcons` 🤔

**Comment by zguolee** [comment]
> > Maybe we can add a new icon component, like [NuxtUi](https://github.com/nuxt/ui/blob/main/packages/ui/src/components/NIcon.vue?rgh-link-date=2022-09-17T08%3A03%3A02Z), so that we do not need to add `inline-block` in every place.
> 
> Yes we can (_IG we might need it in the future_), but why not directly add it in `extraProperties` of `presetIcons` 🤔

I shouldn't have understood before🥲. Adding directly is a very convenient way.
